src/load_image_vectors.py

```diff
@@ -91,9 +91,17 @@ def __init__(self, input_image):
         Initializes CsvImage object, chops of first value (value) as int, saves rest into pixel list
         :param input_image: string of label and intensities, separated by ","
         """
+        # Split input string ("0,0,5 ... 0,0") after each ",", returns a list of strings
         values = input_image.split(",")
+
+        # label (accessible via object_name.label) is set to the first number of the input vector
         self.label = int(values[0])
+
+        # first value is removed from the list because it is now stored as label
+        # -> values list only contains intensity values now
         values.pop(0)
```

Comment:
Performance tip:
Executing `list.pop(0)` results internally into a rewrite of the entire list itself and is, therefore, very slow. You can get around this if you omit the `pop(0)` and use so called slicing in the for loop in line 106.

`for pixel in values[1:]`

This tells the interpreter to only use the list from index 1 onward for the iteration.
https://stackoverflow.com/questions/509211/understanding-slice-notation/509295#509295

Reply:
Done.